pkg/kubelet/dockershim/docker_container.go

```diff
@@ -87,6 +87,10 @@ func (ds *dockerService) CreateContainer(podSandBoxID string, config *runtimeApi
 		}
 	}
 
+	image := ""
+	if iSpec := config.GetImage(); iSpec != nil {
+		image = iSpec.GetImage()
+	}
```

Comment:
In fact I removed the previous comment, because it turned out that `GetImage().GetImage()` is safe.

However, since you have made the change, I think this is more straightforward. Thanks! :)